src/tpm2/crypto/openssl/CryptEccMain.c

```diff
@@ -557,6 +562,11 @@ BnEccGetPrivate(
     UINT32                   orderBytes = BITS_TO_BYTES(orderBits);
     BN_VAR(bnExtraBits, MAX_ECC_KEY_BITS + 64);
     BN_VAR(nMinus1, MAX_ECC_KEY_BITS);
+
+#if USE_OPENSSL_FUNCTIONS_EC
```

Comment:
I think it would be clearer to have two versions of the function, one that is compiled if USE_OPENSSL_FUNCTIONS_EC and the classic one, however code-wise I see no issues.

Reply:
Did this now. Will merge. Thanks for the comments.